mythril/laser/ethereum/instructions.py

```diff
@@ -49,6 +50,154 @@
 keccak_function_manager = KeccakFunctionManager()
 
 
+# opcode -> (min_gas, max_gas)
+OPCODE_COST_FUNCTIONS = {
```

Comment:
Can you move this to a seperate file related to gas consumption?

Reply:
Fixed :+1: